ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -186,10 +188,20 @@ export class Mint extends IronfishCommand {
 
     let assetData
     if (assetId) {
-      assetData = (await client.chain.getAsset({ id: assetId })).content
-      const isAssetOwner = this.isAssetOwner(assetData, accountPublicKey)
-      if (!isAssetOwner) {
-        this.error(`The account '${account}' does not own this asset.`)
+      try {
+        const assetRequest = await client.chain.getAsset({ id: assetId })
+        assetData = assetRequest.content
+        const isAssetOwner = this.isAssetOwner(assetData, accountPublicKey)
+        if (!isAssetOwner) {
+          this.error(`The account '${account}' does not own this asset.`)
+        }
+      } catch (e) {
+        if (e instanceof RpcRequestError && e.code === RPC_ERROR_CODES.NOT_FOUND.valueOf()) {
```

Comment:
This one I could potentially see putting in the client SDK and returning null. nit tho

Reply:
Also don't think we need the `valueOf`